### PR TITLE
include auth_connector resource for tctl sso test instructions

### PR DESCRIPTION
**Pull request by stevenGravy**
`auth_connector` `create` is required for tctl sso test

### CODE REVIEW

File: docs/pages/setup/reference/cli.mdx [hunk 1]
```diff
@@ -1516,7 +1516,7 @@ The test process is safe from side effects in that:
   type="note"
   title="Important"
 >
-  To use this command, you must have access to the `github_request`, `oidc_request`, and `saml_request` resources (depending on the type of connector being tested).
+  To use this command, you must have access to the `github_request`, `oidc_request`, and `saml_request` resources (depending on the type of connector being tested).  Access to the ``auth_connector` `create` is required for all connector types.
```
Comment (Tener) [inline docs/pages/setup/reference/cli.mdx:1519]:
```suggestion
  To use this command, you must have access to the `github_request`, `oidc_request`, and `saml_request` resources (depending on the type of connector being tested).  Access to the `auth_connector` `create` is required for all connector types.
```

**Review by Tener — APPROVED** [review]
Technically connector-level permissions work too (`saml`/`oidc`/`github` instead of all-in-one `auth_connector`), but it may be worth keeping the docs simple.

**Comment by ptgott** [comment]
> Technically connector-level permissions work too (`saml`/`oidc`/`github` instead of all-in-one `auth_connector`), but it may be worth keeping the docs simple.

Since this is a reference, I think it's worth mentioning these other possible permissions as well. Maybe we can split the new sentence into its own paragraph and add another sentence that mentions `saml`, `oidc`, and `github`.

**Comment by zmb3** [comment]
Friendly reminder @stevenGravy - let's try to address feedback in the next 1-2 weeks, otherwise I will consider it deprecated and close.

**Comment by zmb3** [comment]
Closing due to inactivity

**Pull request closed without merging**